polyfills/Object.prototype.toString/test.js

```diff
@@ -0,0 +1,28 @@
+var toString = {}.toString
+
+[
```

Comment:
You might want to do something like `var testData = [...]; testData.forEach(..);`